src/Bicep.Core/Semantics/SemanticDiagnosticVisitor.cs

```diff
@@ -69,7 +69,7 @@ public override void VisitFileSymbol(FileSymbol symbol)
             }
 
             // find instances where the same symbol is imported multiple times under different names
-            foreach (var grouping in symbol.TypeImports.ToLookup(t => (t.TryGetSemanticModel(out var model, out _) ? model : null, t.OriginalSymbolName)).Where(g => g.Count() > 1))
+            foreach (var grouping in symbol.ImportedSymbols.ToLookup(t => (t.TryGetSemanticModel(), t.OriginalSymbolName)).Where(g => g.Key.Item1 is not null && g.Key.OriginalSymbolName is not null && g.Count() > 1))
```

Comment:
This linq is pretty hard to follow. Any ideas for simplifying it?

Reply:
The `.Where(x => Y)` clause can be replaced with a `if (!Y) { continue; }` statement, which would at least unpack this a bit.